view/template/trial.hbs

```diff
@@ -69,6 +69,15 @@
         </div>
         <div class="col-sm-12 col-md-7 col-lg-6 col-xl-6">
             <div class="card">
+                    <form id = "check-compliance" method= "GET">
+                            <label>From date:-</label>
+                                <input id="fromDate"  name="fromDate" type="date" data-format = "yyyy-MM-dd" value="{{ fromDate }}" >
+                            <label>To date:-</label>
+                                <input id="toDate"  name="toDate" type="date" data-format = "yyyy-MM-dd" value="{{ toDate }}">
+                            <div>
+                                <input type = "submit" id = "submit" value = "Check"/>
+                            </div>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/3107513/14943687/6e7461d0-0f94-11e6-87e0-0ae28eb9eb0c.png)

The chart should be on the top.
The two input fields and the button should all be on the same line.
From and To dates should be displaying valid dates.